mock/py/mockbuild/file_downloader.py

```diff
@@ -61,7 +63,7 @@ def get(cls, pkg_url_or_local_file):
                     filed.write(chunk)
             cls.backmap[pkg] = url
             return pkg
-        except Exception as err:  # pylint: disable=broad-except
+        except requests.exceptions.RequestException as err:
```

Comment:
I'm not sure this will work, though.  If you catch the exception, the wrapping decorator can not react on it anymore.

Reply:
I added a handle for all exceptions, I'm not sure if this is possible

Reply:
Thank you for the quick update!

I think there's still the same issue.  When we do `except Exception` inside, the exception is simply swallowed and the external logic (the backoff decorator) can not see it to react appropriately.  I think we could go with two methods, e.g. keep the wrapping `get()` method that could call a backoff-decorated `_get_inner()` method.

Reply:
Thank you for pointing out this issue. I have made modifications to it and I am not sure if it is correct